### DISCUSSION TITLE
Xcode tools problem on macOS Monterey beta 7

### DISCUSSION THREAD

**Discussion by ghost** [Homebrew/discussions#2195] [Getting started] [answered]
Hello all,
I am currently on macOS Monterey beta 7 and using the latest homebrew version (3.2.13-62-97e62066).

However when I try to build formula from source (`-s` option), they all fail with the following error

```
Error: Your command line Tools does not support macOS 12.
It is either outdated or was modified
```

Whereas I just installed the latest command lines tools (September 20) from Apple developer website.

Would you please tell me what I did wrong? Thank you.

**N.B.** When I run `brew config` it clearly says the CLT version is 13.0.0.1.1630607135

**Comment by Bo98** [comment, marked as answer]
CLT 13 RC and final do not contain the macOS 12 SDK, so you'll need to install an older version: CLT 13 beta 5 (or wait for the beta for the next version, which should be soon).
